nsv13/code/modules/munitions/ammunition/broadside_ammo.dm

```diff
@@ -47,6 +57,8 @@
 	var/amount_to_pack = 5
 	var/obj/item/powder_bag/plasma/plasma = FALSE
 	var/obj/item/powder_bag/gunpowder = FALSE
+	var/obj/item/ship_weapon/parts/broadside_load = FALSE
+	var/obj/item/ship_weapon/parts/broadside_load/nuclear = FALSE
```

Comment:
```suggestion
	var/nuclear = FALSE
```
These variables are only used as a boolean, so you can just have one nuclear variable and check if(nuclear) instead of checking it.
For the event that there is not yet any type of shell in there, you can also make a unique case if(!load_amount), which then sets nuclear to FALSE or TRUE based on the first type of shell loaded.